docs/_data/projects.yml

```diff
@@ -1,4 +1,3 @@
-- docker/infrakit
```

Comment:
Should also replace https://github.com/moby/mobywebsite/blob/cd3247d47f6c396c49bc0b534149afffe5869dd2/docs/_data/projects.yml#L15

with `distribution/distribution`

Reply:
updated 👍